### DISCUSSION TITLE
Exisiting project router replacement questions

### DISCUSSION THREAD

**Discussion by Gildedter** [roxiness/routify#482] [Q&A]
I have an SPA project based on [this](https://github.com/firtadokei/ci-svelte-appstarter) that's currently using `svelte-spa-router`, in the the [docs ](https://www.routify.dev/guide/introduction/structure) it says its expecting a folder named `src`, currently mine's called `resource` (defined by my dependency), is it possible to use this as an alternative router? And how hard would it be to integrate on an existing (specifically the template I'm using which is a PHP backend) project?

**Comment by jakobrosenberg** [comment]
Hi @Gildedter , sorry about the late followup. It's been a hectic week.

It shouldn't be a problem integrating Routify at all. The only requirement is that your files/folders are organized in the same structure you want for your navigation.

I would recommend you to visit us on Discord as it's likely easier to assist you in the chat.
